lib/inits/migrations/Migration.ts

```diff
@@ -1,27 +1,28 @@
 import { ManifestVersion } from "../../manifest/ManifestTypes"
 import { Effects } from "../../types"
+import { Utils } from "../../util"
 
-export class Migration<Version extends ManifestVersion> {
+export class Migration<WD, Version = ManifestVersion> {
```

Comment:
Why did you remove the manifest version extends and did a default, this could be dangerous